iFixit/react-commerce
# Integrate with Strapi

This PR closes #75 and #67.

- Refactors data fetching to use Strapi API
- Refactors collection components into sections
- Moves fetching of related posts client side to reduce initial page latency 

## QA

1. Visit the Vercel PR preview
2. Visit collection page and verify that collection uses data from Strapi
